packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -105,6 +105,9 @@ declare global {
 
 const SEARCH_MAX_SAVED_RESULTS = 10
 const SEARCH_MAX_TOTAL_RESULTS = 50
+export const AUDIO_DECIMALS = 18
```

Comment:
use fixed decimal for this please!

Reply:
fixed decimal constructors: if you pass in string, it treats it as uiAmount and looks for decimal place, then multiplies by 10**decimal places. if you pass in a bigint, it treats it as the raw wei value. this was the gotcha that got me good.